### PR TITLE
Added Prims Algorithm for MST(UPDATED)

### PR DESCRIPTION
**Pull request by Subhojit-Kumar-Saha**
/***** Prim's Algorithm for finding Minimum Spanning Tree *****/

Sample Inputs are already given :
graph[V][V] = { { 0, 2, 0, 6, 0 },
{ 2, 0, 3, 8, 5 },
{ 0, 3, 0, 0, 7 },
{ 6, 8, 0, 0, 9 },
{ 0, 5, 7, 9, 0 } };

Sample Output:

Edge Value
1 - 0 2
2 - 1 3
3 - 0 6
4 - 1 5

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
Please add this description sample input output in the code in the comments section...
Common, it's a simple thing. You should have done it by now

**Comment by Subhojit-Kumar-Saha** [comment]
Description added in the comment section

File: prims_algorithm.cpp [hunk 1]
```diff
@@ -44,3 +55,12 @@ int main()
 }
 
 
+/*
```
Comment (Bhavnaharitsa) [inline prims_algorithm.cpp]: Remove this */ one is enough

Comment (Subhojit-Kumar-Saha) [reply prims_algorithm.cpp]: this one is for sample output description

Comment (Subhojit-Kumar-Saha) [reply prims_algorithm.cpp]: /*
Sample Output:
Edge  Value
1 - 0   2
2 - 1   3
3 - 0   6
4 - 1   5
*/

**Comment by Subhojit-Kumar-Saha** [comment]
@Bhavnaharitsa  please check

File: prims_algorithm.cpp (hunk 1 above)
Comment (Bhavnaharitsa) [reply prims_algorithm.cpp]: push the input output on top

Comment (Subhojit-Kumar-Saha) [reply prims_algorithm.cpp]: Done

**Comment by Bhavnaharitsa** [comment]
There are two prims algorithms
Which one do you want me to check?

**Comment by Subhojit-Kumar-Saha** [comment]
where?

**Comment by Subhojit-Kumar-Saha** [comment]
latest commit

**Comment by Bhavnaharitsa** [comment]
https://github.com/NJACKWinterOfCode/nwoc_algorithms/pull/46
This is a Prims algorithm too

**Comment by Subhojit-Kumar-Saha** [comment]
No no not #46

**Comment by Subhojit-Kumar-Saha** [comment]
#83 is latest one

**Pull request merged**